tools/tests/unit/test_spice_examples.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+import spiceypy as spice
+
+from tools.spice.spice_examples import (
+    build_annotated_events,
+    get_attitude_timerange,
+    get_particle_velocity,
+)
+from tools.spice.spice_utils import (
+    list_files_with_extensions,
+)
+
+
+@pytest.fixture()
+def kernel_directory():
+    """Kernel directory."""
+    kernel_directory = Path(__file__).parents[1] / "test_data" / "spice"
+    return kernel_directory
+
+
+@pytest.fixture()
+def kernels(kernel_directory):
+    """Loads all kernels."""
+    kernels = list_files_with_extensions(
+        kernel_directory, [".tsc", ".tls", ".tf", ".bsp"]
+    )
+    return kernels
+
+
+@pytest.fixture()
+def direct_events():
+    """Loads direct events test data.
+
+    Possible example of future direct_events structure:
+    direct_events.keys()
+    dict_keys(['eventID', 'tdb', 'xstart', 'ystart', 'xstop', 'ystop',
+    'eventtype', 'ph', 'spinphase', 'tof', 'ishead90', 'yf', 'd',
+    'energy', 'rvecultra', 'vultra', 'rmagultra', 'rnormultra'])
+    """
+    direct_events = {
+        "vultra": np.array(
+            [-5.7718470e08, -8.8437594e08, -9.5545958e08], dtype=np.float32
+        ),
+        "tdb": 802100000.0,
+    }
+
+    return direct_events
+
+
+def test_get_attitude_timerange(kernels, kernel_directory):
+    """Tests get_attitude_timerange function."""
+
+    spice.kclear()
+
+    spice.furnsh(kernels)
```

Comment:
If there is an error in this routine, or we run tests in multiple processes in the future this will cause issues. We need to put this in an `with spice.Kernels():` call to properly clean up if issues arise during these tests.